nodogsplash2/files/nodogsplash.config

```diff
@@ -14,27 +14,34 @@ config nodogsplash
 
   # The network the users are connected to
   option network 'lan'
-  option gatewayname 'LEDE Nodogsplash'
+  option gatewayname 'OpenWrtLEDE Nodogsplash'
```

Comment:
We can use OpenWrt only now, LEDE is in the process to be merged back into OpenWrt (depending on the point of view)